src/components/Nodes/GethConfig/index.js

```diff
@@ -63,7 +99,31 @@ class GethConfig extends Component {
       return null
     }
 
-    return <StyledError>{errorMessage}</StyledError>
+    return (
+      <SnackbarContent
```

Comment:
nit: this would be nice to extract as a shared notification/toast component, so that all the mui junk isn't required everytime we want to use one. Instead just pass in the type, message, and dismiss function, e.g. `<Notification type="error" message={message} dismiss={dismissFunc} />`, or
`<Notification type="warning" message={message} dismiss={dismissFunc} />`. Idk if `Notification` is the best name, but I like it more than "Snackbar" 😅

Reply:
thanks, done!